MarsTestAutomation/SpecFlow/SellerProfile.Education.feature

```diff
@@ -0,0 +1,88 @@
+Feature: SellerProfile_Education
+As a Seller
+I want to add Education in my Profile Details
+So that
+The people looking my profile can see what Education i done.
+    
+# profile Education Added
+    @sellerprofiletest
+    Scenario: Add Profile Education with data
+        Given I logged into Trade Skills portal successfully
+        And I click on Education and Add New button
+        When I Enter seller Education details and click on Add button
+        Then Education details should be Added and saved successfully
+
+    @sellerprofiletest
+    Scenario: Add Profile Education without data
+        Given I logged into Trade Skills portal successfully
+        And I click on Education and Add New button
```

Comment:
i would put the and add new button in a new line as you can see it starts with and, please change every where!

Reply:
done